benefits-italy.md

```diff
@@ -0,0 +1,6 @@
+# Benefits
+
+1. Good food
+2. Good weather (sometimes)
+3. The sea
+4. Pizza
```

Comment:
Hi Edoardo, thank you for your effort in creating this list. I do agree with all of the points, but I feel like pizza is already part of number 1. Good food. Could you revise this? Thank you

Reply:
Oh that's true. But it is so good that it deserves its own point. Are you sure I should delete it?